### DISCUSSION TITLE
Is there a common errors setting to fix uppercase I's mixed with lowercase L's?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5144] [General]
![image](https://user-images.githubusercontent.com/82276985/122781513-e2a2d800-d27d-11eb-8253-de61656db242.png)

**Comment by niksedk** [comment]
You could try "Fix common OCR errors":
![image](https://user-images.githubusercontent.com/1261331/122782216-d3894d80-d2b0-11eb-8611-4107fcb14a54.png)

**Comment by DrReddP** [comment]
Balls, I had the other OCR one selected. Thank you

**Reply by niksedk** [reply]
lol, you're welcome :)
